Week_05/state_names.py

```diff
@@ -0,0 +1,66 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+
+#   Version one without functions
+# CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+#                 "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+#
+# state_code = input("Enter short state:   ").upper()  # added .upper() to make the input uppercase
+# while state_code != "":
+#     if state_code in CODE_TO_NAME:
+#         print(state_code, "is", CODE_TO_NAME[state_code])
+#     else:
+#         print("Invalid short state")
+#     state_code = input("Enter short state:").upper()  # added .upper() to make the input uppercase
+
+# CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+#                 "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+
+
+# Version two with functions
+# def main():
+#     state_code_to_name = get_state_code_to_name()
+#     state_code = input("Enter short state:   ").upper()  # added .upper() to make the input uppercase
+#     while state_code != "":
+#         if state_code in state_code_to_name:
+#             print(state_code, "is", state_code_to_name[state_code])
+#         else:
+#             print("Invalid short state")
+#         state_code = input("Enter short state:").upper()
+#
+#
+# def get_state_code_to_name():
```

Comment:
Unsure if this function is more effective than just keeping dictionary in main as it is :)

Reply:
I agree! just wanted to explore different option :)